File: Cpp/STL/1768.py
word1 = "abccccccccccccccc"
word2 = "pqrstuvw"

if len(word1)<len(word2):
    length = len(word1)
else:
    length = len(word2)

ans = ""
for i in range(length):
    ans+=word1[i] + word2[i]
    if i==length-1:
        if length!=len(word2):
            ans+=word2[i+1:]
        if length!=len(word1):
            ans+=word1[i+1:]

print(ans)

File: Cpp/STL/1768.cpp
#include <bits/stdc++.h>
using namespace std;

int main()
{
    string word1 = "abccccccccccccccc", word2 = "pqrstuvw";
    int length = word1.length() < word2.length() ? word1.length() : word2.length();

    string ans = "";
    for (int i = 0; i < length; i++)
    {
        ans.push_back(word1.at(i));
        ans.push_back(word2.at(i));
        if (i == length - 1)
        {
            if (length != word2.length())
                ans += word2.substr(i + 1);
            if (length != word1.length())
                ans += word1.substr(i + 1);
        }
    }
    cout << ans;
}